code/game/objects/effects/projectile/projectile_effects.dm

```diff
@@ -0,0 +1,54 @@
+/obj/effect/projectile
+	name = "pew"
+	icon = 'icons/obj/projectiles.dmi'
+	icon_state = "nothing"
+	layer = 4.5
```

Comment:
what's with the magical layer?

Reply:
I have no idea what this layer is for, but I assume it's important.